frontend/packages/console-shared/src/components/form-utils/FormFooter.scss

```diff
@@ -2,6 +2,8 @@
 
 .ocs-form-footer {
   padding: var(--pf-t--global--spacer--md) $pf-v6-global-gutter;
+  z-index: 101; // bump above --pf-v6-c-menu-toggle--border--ZIndex (100) to prevent menu borders from appearing in the footer
```

Comment:
is there a different z-index token we can use instead?

Reply:
Updated with the token that the class resolves to